packages/@aws-cdk/aws-glue-alpha/README.md

```diff
@@ -42,8 +42,9 @@ new glue.Job(this, 'ScalaSparkEtlJob', {
     glueVersion: glue.GlueVersion.V4_0,
     script: glue.Code.fromBucket(bucket, 'src/com/example/HelloWorld.scala'),
     className: 'com.example.HelloWorld',
-    extraJars: [glue.Code.fromBucket(bucket, 'jars/HelloWorld.jar')],
+    extraJars: [glue.Code.fromBucket(bucket, 'jars/HelloWorld.jar')]
```

Comment:
These kinds of commas are not typos, it makes it easier to extend without having to worry if it's the last line or higher up.